client/wp-admin-scripts/payment-method-promotions/payment-promotion-row.scss

```diff
@@ -29,7 +29,7 @@
 	.pre-install-wcpay_name {
```

Comment:
I think this class name needs to be updated here and inside the `PaymentPromotionRow` component.